boards/Pynq-Z1/base/notebooks/register_map_intro.ipynb

```diff
@@ -0,0 +1,332 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Introduction to the Register Map\n",
+    "\n",
+    "In PYNQ 2.4 it is now possible to access the register map for IP in the overlay directly. This functionality requires that the `.hwh` file is distributed along with the bitstream. This can be generated from Vivado.\n",
```

Comment:
I think we can de-emphasize the version number since we will have to update this again in next release.